src/devices/tuya.ts

```diff
@@ -8734,6 +8734,66 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('NAS-WV03B', ['_TZE204_rzrrjkz2']),
```

Comment:
```suggestion
        zigbeeModel: ['NAS-WV03B'],
```

The modelId is unique, so no need to use fingerprint